StortSpelProjekt/Engine/src/Particles/ParticleEffect.h

```diff
@@ -18,7 +19,7 @@ class Mesh;
 
 struct RandomParameter
 {
-	float2 interval = { 0.0f, 1.0f };
+	float2 interval = { 0.0f, 1.0f }; // default should be 0,0 rand does not work with 0,0
 	//rand distribution
 };
```

Comment:
No need to have a struct with only 1 variable in it. But since you said that it will hold more data in the future, it is ok